documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -895,13 +895,16 @@ engines running on the JUnit Platform via one of the following mechanisms.
     `systemProperty` or `systemProperties` DSL.
   * <<running-tests-build-maven-config-params,Maven Surefire provider>>: use the
     `configurationParameters` property.
-2. JVM system properties.
-3. The JUnit Platform configuration file: a file named `junit-platform.properties` in the
+2. The `configurationParametersResource()` and `configurationParametersResources()`
+  methods in the `LauncherDiscoveryRequestBuilder` which are used to select a `.properties`
+  file on the classpath with configuration parameters.
```

Comment:
```suggestion
2. The `configurationParametersResources()` method in the `LauncherDiscoveryRequestBuilder`
  which is used to specify one or more Java `Properties` files in the class path that
  provide configuration parameters.
```

The `configurationParametersResource()` method no longer exists -- right?

Reply:
Correct, fixed